testproject/Assets/StreamingAssets/.gitignore

```diff
@@ -0,0 +1,2 @@
+/BuildInfo.json
```

Comment:
There's already 3+ .gitignores in the repo can we add this to one of those instead of this deeply nested one?

Reply:
so, let's revert this file?

Reply:
wtf... I did delete it... why is it still here?